benchmarks/README.md

```diff
@@ -0,0 +1,50 @@
+# JetStream Benchmark And Eval
+
+## Install Dependencies 
+
+```
+cd ~/JetStream/Benchmark
+pip install -r requirements.in
+```
+
+## Benchmark 
+
+### Prepare DataSet
+
+```
+cd ~/data
+wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
+
+``` 
+
+### Run Benchmark with maxtext tokenizer
+
+```
+python Benchmark/benchmark_serving.py \
```

Comment:
maybe directly call benchmark_serving.py if the user is already in /benchmarks, or use JetStream/benchmarks/benchmark_serving.py

Reply:
sounds good, updated.